src/core/utils.py

```diff
@@ -0,0 +1,22 @@
+from src.core.common_path import CommonPaths
+
+
+class ConfigLoader:
```

Comment:
@vasylboliuk it is more common to hold configuration specific logic close to the configuration module or package. Utils is used mostly for the functions and classes that can be used accross all modules/classes and this one, apperenlty, is going to be used by config management only,